website/docs/r/vdc_group.html.markdown

```diff
@@ -0,0 +1,93 @@
+
+## Example Usage
+
+```hcl
+data "vcd_org_vdc" "startVdc" {
+  name = "existingVdc"
+}
+
+data "vcd_org_vdc" "additionalVdc" {
+  name = "oneMoreVdc"
+}
+
+resource "vcd_vdc_group" "new-vdc-group" {
+  org                   = "myOrg"
+  name                  = "newVdcGroup"
+  description           = "my description"
+  starting_vdc_id       = data.vcd_org_vdc.startVdc.id
+  participating_vdc_ids = [data.vcd_org_vdc.startVdc.id, data.vcd_org_vdc.additionalVdc.id]
+  dfw_enabled           = true
+  default_policy_status = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organizations
+* `name` - (Required)  - The name for VDC group
+* `description` - (Optional)  - VDC group description
+* `starting_vdc_id` - (Required)  - With selecting a starting VDC you will be able to create a group in which this VDC can participate. **Note**. It must be included in `participating_vdc_ids` to participate in this group.
+* `participating_vdc_ids` - (Required)  - The list of organization VDCs that are participating in this group. **Note**. `starting_vdc_id` isn't automatically included.
+* `dfw_enabled` - (Optional)  - Whether Distributed Firewall is enabled for this VDC Group.
+* `default_policy_status` - (Optional)  - Whether this security policy is enabled. `dfw_enabled` must be `true`.
+
+## Attribute Reference
+
+The following attributes are exported on this resource:
+
+* `id` - The VDC group ID
+* `error_message` - More detailed error message when VDC group has error status
+* `local_egress` - Status whether local egress is enabled for a universal router belonging to a universal VDC group.
+* `network_pool_id` - ID of used network pool.
+* `network_pool_universal_id` - The network provider’s universal id that is backing the universal network pool.
+* `network_provider_type` - Defines the networking provider backing the VDC Group.
+* `status` - The status that the group can be in (e.g. 'SAVING', 'SAVED', 'CONFIGURING', 'REALIZED', 'REALIZATION_FAILED', 'DELETING', 'DELETE_FAILED', 'OBJECT_NOT_FOUND', 'UNCONFIGURED').
+* `type` - Defines the group as LOCAL or UNIVERSAL.
+* `universal_networking_enabled` - True means that a VDC group router has been created.
+* `participating_org_vdcs` - A list of blocks providing organization VDCs that are participating in this group details. See [Participating Org VDCs](#participatingOrgVdcs) below for details.
+
+<a id="participatingOrgVdcs"></a>
+## Participating Org VDCs
+
+* `vdc_id` - VDC ID.
+* `vdc_name` - VDC name.
+* `site_id` - Site ID.
+* `site_name` - Site name.
+* `org_id` - Organization ID.
+* `org_name` - Organization Name.
```

Comment:
After some fields you write `Name`, after others - `name`. Please fix consistency.

Reply:
changed